wpilibc/src/main/native/include/frc/simulation/DifferentialDrivetrainSim.h

```diff
@@ -55,11 +62,18 @@ class DifferentialDrivetrainSim {
    * @param wheelRadius The radius of the wheels on the drivetrain.
    * @param trackWidth  The robot's track width, or distance between left and
    * right wheels.
+   * @param measurementStdDevs Standard deviations for measurements, in the form [heading, left velocity,
+   *                           right velocity, left distance, right distance]^T. Can be omitted if no noise is desired.
+   *                           Gyro standard deviations of 0.0001 radians, velocity standard deviations of 0.05 m/s,
+   *                           and position measurement standard deviations of 0.005 meters are a reasonable starting
+   *                           point.
    */
   DifferentialDrivetrainSim(frc::DCMotor driveMotor, double gearing,
                             units::kilogram_square_meter_t J,
                             units::kilogram_t mass, units::meter_t wheelRadius,
-                            units::meter_t trackWidth);
+                            units::meter_t trackWidth,
+                            const std::array<double, 5>& measurementStdDevs =
+                      std::array<double, 5>{});
```

Comment:
```suggestion
                      std::array<double, 5>{0.0, 0.0, 0.0, 0.0, 0.0});
```
Not zeroing these out causes UB.